paddle/fluid/imperative/layer.cc

```diff
@@ -245,6 +255,21 @@ OpBase::OpBase(size_t id, const framework::OpDesc& op_desc,
   VLOG(3) << "Construct Op: " << op_desc.Type() << std::endl;
 }
 
+bool OpBase::create_operator_base() {
+  const auto& info = framework::OpInfoMap::Instance().Get(type_);
+  if (info.Checker() != nullptr) {
+    info.Checker()->Check(&attrs_);
+  }
+
+  auto input_name_map = CreateVarNameMap(info, type_, ins_, true);
+  auto output_name_map = CreateVarNameMap(info, type_, outs_, false);
+  op_ = framework::OpRegistry::CreateOp(type_, std::move(input_name_map),
+                                        std::move(output_name_map),
+                                        std::move(attrs_));
```

Comment:
Move `attrs_` is a bit strange, since it is a member of `OpBase`. After `create_operator_base` is invoked, the fact that `attrs_` would be changed is confused to developers.